content/blog/2021-12-24-darktable-3.8/index.md

```diff
@@ -10,7 +10,8 @@ tags:
   - darktable-release
 ---
 
-_Translations of this article: [German](https://www.bilddateien.de/blog/2021-12-24-darktable-3-8-tastaturbefehle-ueberall.html)._
+_Translations of this article: [German](https://www.bilddateien.de/blog/2021-12-24-darktable-3-8-tastaturbefehle-ueberall.html), 
+[Espanol](https://www.ramonclemente.com/blog/post/darktable-38-lanzamiento-invierno-2021)._
```

Comment:
For consistency we should probably say "Spanish" rather than "Espanol"

Reply:
ok, no problem.
To be honest I just copied this from my german version where I use the native names of those languages.
Will see if I am able to correct this ...